docs/assertions/assertions.md

```diff
@@ -30,6 +30,167 @@ with(process.out.imputed_plink2) {
 }
 ```
 
+## Comparing Channels with Using `assertInAnyOrder`
```

Comment:
Thanks for all the examples and documentation! :+1:

Reply:
For sure! Feels like the solution in the end of the day was realising how to better leverage groovy =]